bazel/repositories.bzl

```diff
@@ -1006,6 +1006,9 @@ def _com_github_luajit2_luajit2():
     external_http_archive(
         name = "com_github_luajit2_luajit2",
         build_file_content = BUILD_ALL_CONTENT,
+        patches = ["@envoy//bazel/foreign_cc:luajit2.patch"],
```

Comment:
Are you sure? I think this patch is already present on 2.2

Reply:
It is currently not present in 2.2: some luajit2 changes have not been merged from 2.1 into 2.2. This PR contains all missing luajit2 code that is needed to build envoy/proxy with luajit2.

Reply:
True. I just hit this in a ppc product build. The build fails because of this.

Reply:
@jwendell just to be clear, do you mean that you want this left in?